src/Domain/CoordinateMaps/TimeDependent/Shape.hpp

```diff
@@ -93,49 +86,95 @@ namespace domain::CoordinateMaps::TimeDependent {
  * FunctionOfTime representation we are using, because the resulting expressions
  * are much shorter.
  *
- * An additional domain-dependent transition function
+ * An additional dimensionless domain-dependent transition function
  * \f$f(r, \theta, \phi)\f$ ensures that the distortion falls off correctly to
- * zero at the boundary of the domain.
+ * zero at a certain boundary (must be a block boundary). The function \f$f(r,
+ * \theta, \phi)\f$ is restricted such that
+ *
+ * \f{equation}{
+ * 0 \leq f(r, \theta, \phi) \leq 1
+ * \f}
  *
  * ### Mapped coordinates
  *
+ * Given a point with cartesian coordinates \f$\xi^i\f$, let the polar
+ * coordinates \f$(r, \theta, \phi)\f$ with respect to a center \f$x_c^i\f$ be
+ * defined in the usual way:
+ *
+ * \f{align}{
+ * \xi^0 - x_c^0 &= r \sin(\theta) \cos(\phi)\\
+ * \xi^1 - x_c^1 &= r \sin(\theta) \sin(\phi)\\
+ * \xi^2 - x_c^2 &= r \cos(\theta)
+ * \f}
+ *
  * The shape map maps the unmapped
  * coordinates \f$\xi^i\f$ to coordinates \f$x^i\f$:
  *
  * \f{equation}{
- * x^i = \xi^i - (\xi^i - x_c^i) f(r, \theta, \phi) \sum_{lm}
+ * x^i = \xi^i - (\xi^i - x_c^i) \frac{f(r, \theta, \phi)}{r} \sum_{lm}
  * \lambda_{lm}(t)Y_{lm}(\theta, \phi).
  * \f}
  *
+ * Or written another way
+ *
+ * \f{equation}{\label{eq:map_center_plus_distortion}
+ * x^i = x_c^i + (\xi^i - x_c^i) \left(1 - \frac{f(r, \theta, \phi)}{r}
+ * \sum_{lm} \lambda_{lm}(t)Y_{lm}(\theta, \phi)\right).
+ * \f}
+ *
+ * The form in Eq. \f$\ref{eq:map_center_plus_distortion}\f$ makes two things
+ * clearer
+ *
+ * 1. This shape map is just a radial distortion about \f$x_c^i\f$
+ * 2. The coefficients \f$\lambda_{lm}\f$ have units of distance because
+ *    \f$\sum\lambda_{lm}(t)Y_{lm}(\theta,\phi) / r\f$ must be dimensionless
+ *    (because \f$f\f$ is dimensionless).
+ *
  * ### Inverse map
  *
  * The inverse map is given by:
  * \f{equation}{
  * \xi^i = x_c^i + (x^i-x_c^i)*(r/\tilde{r}),
  * \f}
  * where \f$\tilde{r}\f$ is the radius of \f$\xi\f$, calculated by the
- * transition map. For more details, see
- * ShapeMapTransitionFunction::original_radius_over_radius .
+ * transition map. In order to compute \f$r/\tilde{r}\f$, the following equation
+ * must be solved
+ *
+ * \f{equation}{
+ * \frac{r}{\tilde{r}} =
+ * \frac{1}{1-f(r,\theta,\phi)\sum\lambda_{lm}(t)Y_{lm}(\theta,\phi) / r}
+ * \f}
+ *
+ * For more details, see
+ * \link
+ * domain::CoordinateMaps::ShapeMapTransitionFunctions::ShapeMapTransitionFunction::original_radius_over_radius
+ * ShapeMapTransitionFunction::original_radius_over_radius \endlink.
```

Comment:
Wasn't sure what to do about this. Even without the `\link` it would be over 80 characters.

Reply:
Whitelisting `\link` in general seems like the best approach and is helpful in the future as well.